### DISCUSSION TITLE
Keybind to increase the font size?

### DISCUSSION THREAD

**Discussion by rochacbruno** [lapce/lapce#1597] [Q&A] [answered]
Hi,

Is there a keybinding to increase the font size like ctrl++ or mouse wheel?

**Comment by panekj** [comment, marked as answer]
Nope

**Comment by rochacbruno** [comment]
Hi @panekj thanks for the answer,

Is that feature on the roadmap? or that must come as a plugin? I am willing on working on that.

I plan to use the editor for recording training classes and zoom in the terminal is needed.

**Reply by panekj** [reply]
https://github.com/lapce/lapce/issues/785
